Emby.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -234,36 +230,36 @@ private static string NormalizePath(string path, IFileSystem fileSystem)
             return tempPath;
         }
 
-        private static SKCodecOrigin GetSKCodecOrigin(ImageOrientation? orientation)
+        private static SKEncodedOrigin GetSKCodecOrigin(ImageOrientation? orientation)
```

Comment:
Should this method be renamed `GetSKEncodedOrigin` to match the change in type name?